pallets/api/src/messaging/mod.rs

```diff
@@ -273,6 +279,10 @@ pub mod pallet {
 			/// The messages which were removed.
 			messages: Vec<MessageId>,
 		},
+		/// An ISMP message has timed out.
+		IsmpTimedOut { commitment: H256 },
+		/// An XCM message has timed out.
+		XcmTimedOut { query_ids: Vec<QueryId> },
```

Comment:
I think this could use a rework! Issue is that it's a singular event, but has a plural number of items.
a few thoughts / different options:
- Rename to "XcmsTimedOut" (or something)
- create two events: a singular xcm timed out, and a many xcm timed out
- Keep it as a sinular event, but then you have to emit an event for each timed out xcm